students/jungchan/users/views.py

```diff
@@ -1,3 +1,39 @@
 from django.shortcuts import render
 
 # Create your views here.
+import json, re
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import UserInformation
+
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
```

Comment:
```suggestion
        try:
            data = json.loads(request.body)
```
data를 받는 부분도 Try안에 넣어주면 좋을 것 같습니다.